src/main/scala/chisel3/iotesters/SteppedHWIOTester.scala

```diff
@@ -132,30 +132,30 @@ abstract class SteppedHWIOTester extends HWIOTester {
   private def createVectorsAndTestsForOutput(output_port: Data, counter: Counter): Unit = {
     val output_values = Vec(
       test_actions.map { step =>
-        output_port.cloneType.fromBits(UInt(step.output_map.getOrElse(output_port, BigInt(0))))
+        output_port.cloneType.fromBits((step.output_map.getOrElse(output_port, BigInt(0))).asUInt)
       }
     )
     val ok_to_test_output_values = Vec(
       test_actions.map { step =>
-        Bool(step.output_map.contains(output_port))
+        (step.output_map.contains(output_port)).asBool
       }
     )
 
     when(ok_to_test_output_values(counter.value)) {
-      when(output_port.toBits() === output_values(counter.value).toBits()) {
+      when(output_port.asUInt() === output_values(counter.value).asUInt()) {
```

Comment:
I don't think we need the empty parentheses for asUInt, just use like `output_port.asUInt`?